### ISSUE TITLE
Seek values are different for D-Pad and Fast Forward/Rewind buttons

### ISSUE THREAD

**Issue by silentTeee** [jellyfin/jellyfin-androidtv#3033]
### Describe the bug

Not sure if this was intentional, but the directional arrow buttons on my television's remote seem to have very different seek behaviors than the traditional fast-forward and rewind buttons. 

When I use the dedicated Fast Forward and Rewind buttons on my remote, they seem to honor the step values set on my Jellyfin server, and will increment exactly to the current position +/- the step. But when I use the remote's D-Pad Left and Right buttons, they seem to seek to the nearest 30 second interval (if I am at position 00:12:45 and I push Left, it will jump back to 00:12:30, and if I push left again it will jump to 00:12:00).

Since not all TV remotes offer dedicated playback buttons and sometimes just map them to the D-Pad (with the 'Select' button usually inheriting Play/Pause behavior), I am wondering if it makes sense to have behavior be consistent across both button sets.

### Application version

0.15.12

### Device information

HiSmartTV A4

### Android version

Android 9

### Jellyfin server version

10.8.10

**Comment by nielsvanvelzen** [comment]
This is intended behavior for now. It might be revisited when the video player is redesigned as a part of #1057.

**Comment by silentTeee** [comment]
Understood, should I leave this issue open then, or would you prefer it closed?

**Comment by nielsvanvelzen** [comment]
We can keep the issue open

**Comment by rawkhopper** [comment]
I have a TCL TV and the remote only has d pads. I would like them to be 10 seconds as well. Maybe everyone would be happy if it were configurable? Then each person could choose 10sec or 30sec? I totally agree that 30secs is way too long when you miss a sentence or two.  I tend to forget what I was listening for by the time 30secs plays back :). Then I end up doing it again!

**Comment by FunctionDJ** [comment]
I frequently rewind shows or movies, e.g. because i didn't understand some dialogue.

It would be incredibly helpful if there were settings to change the seek/jump length and the seek/jump type (prev/next interval vs. relative).

Also a setting to toggle autoresume after seek/jump would be useful. I think it pauses for the purpose of not lagging while seeking, but resume could simply be debounced with something like 500ms or 1s of delay.

**Comment by ericleb010** [comment]
@nielsvanvelzen Any chance you would be open to at least [set the `SEEK_LENGTH` to `10000`](https://github.com/jellyfin/jellyfin-androidtv/blob/master/app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/CustomSeekProvider.kt#L11) for the time being? If this will be revisited at some point in the (unspecified) future, I think a more granular setting would be appreciated all around.

**Comment by nielsvanvelzen** [comment]
Yeah that's fine, I'd accept a pull request for that. There's also #3794 but that appears to be abandoned.

**Comment by ericleb010** [comment]
I'm okay with the idea of a local configuration, but ultimately this should just approach seeking like `fastForward` and `rewind` do, with `getSafeSeekPosition` or something like that. I see that they behave differently though, so I'm not sure how difficult that makes harmonizing them.
